### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by bensternthal**
Added template for 2024 Google Season Of Docs

### CODE REVIEW

**Comment by bensternthal** [comment]
@ovflowd @joesepi hoping you can do a quick review on this so we can merge and open up the call for ideas.

**Review by joesepi — APPROVED** [review]

**Comment by bensternthal** [comment]
@tobie or @ovflowd can we get a review?

**Review by ovflowd — APPROVED** [review]
LGTM!!

File: mentorship/2024/google-season-of-docs/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+```md
+### Project Idea: {project name}
+
+#### Problem
+
+#### How would we measure success?
+
+#### What skills would a technical writer need to work on this project?
+
+####Expected outcomes:
```
Comment (ovflowd) [inline mentorship/2024/google-season-of-docs/README.md:28]: Oh missing space

**Comment by ovflowd** [comment]
> @ovflowd @joesepi hoping you can do a quick review on this so we can merge and open up the call for ideas. 

Conpletely missed this one 🙈

**Pull request merged**